content/intro-to-storybook/vue/en/simple-component.md

```diff
@@ -79,7 +79,7 @@ export const Default = () => ({
   template: taskTemplate,
   props: {
     task: {
-      default: taskData,
+      default: () => (taskData),
```

Comment:
@sshiling we can do without the extra parenthesis, to avoid having prettier adding unnecessary warnings. I've just checked and all three modes work without it.

Reply:
@jonniebigodes okay. Thank you for your detailed explanations.